pkg/services/sqlstore/migrations/annotation_mig.go

```diff
@@ -123,7 +123,21 @@ func addAnnotationMig(mg *Migrator) {
 	mg.AddMigration("Make epoch_end the same as epoch", NewRawSqlMigration("UPDATE annotation SET epoch_end = epoch"))
 	mg.AddMigration("Move region to single row", &AddMakeRegionSingleRowMigration{})
 
-	// TODO! drop region_id column?
+	mg.AddMigration("Remove index org_id_epoch on annotation table", NewDropIndexMigration(table, &Index{
+		Cols: []string{"org_id", "epoch"}, Type: IndexType,
+	}))
+
+	mg.AddMigration("Remove index org_id_dashboard_id_panel_id_epoch on annotation table", NewDropIndexMigration(table, &Index{
+		Cols: []string{"org_id", "dashboard_id", "panel_id", "epoch"}, Type: IndexType,
+	}))
+
+	mg.AddMigration("Add index for org_id_dashboard_id_epoch_epoch_end on annotation table", NewAddIndexMigration(table, &Index{
+		Cols: []string{"org_id", "dashboard_id", "epoch", "epoch_end"}, Type: IndexType,
+	}))
```

Comment:
Have you tested this against an index with epoch_end before epoch?

Reply:
We're working on it why this is a WIP. Do you mean having the epoch_end before epoch in the index?

Reply:
Yes, maybe I wasn't clear in the issue.  The query looks for rows with `epoch_end > From` and `epoch < To`, in most cases there will be far fewer rows with `epoch_end > From` than `epoch < To` so the index will be more efficient if `epoch_end` is before `epoch`.